http-add-on/README.md

```diff
@@ -82,12 +82,11 @@ their default values.
 
 | Parameter                                                  | Description                               | Default                                         |
 |:-----------------------------------------------------------|:------------------------------------------|:------------------------------------------------|
-| `version`                                                  | Image tag for the http add on             | The latest release                              |
+| `images.tag`                                               | Image tag for the http add on             | `latest`                              |
```

Comment:
Since we'd prefer it to default to the app version, we should just leave this empty here
```suggestion
| `images.tag`                                               | Image tag for the http add on             | None, it used Helm chart's app version as a default                              |
```